src/user-guide/writing-workflows/scheduler.rst

```diff
@@ -10,53 +10,110 @@ workflows.
 
 
 .. _user_guide.scheduler.workflow_event_handling:
+.. _user_guide.scheduler.workflow_events:
 
-Workflow Event Handling
------------------------
-
-Workflow event handlers allow configured commands to run when workflow events
-occur.
-
-.. note::
+Workflow Events
+---------------
 
-   Cylc supports workflow events e.g. ``startup`` and ``shutdown``
-   and task events e.g. ``submitted`` and ``failed``.
+Cylc supports workflow events which pertain to the :term:`scheduler` e.g.
+``startup`` and ``shutdown`` and task events e.g. ``submitted`` and ``failed``
+which pertain to :term:`tasks <task>`. This section covers workflow events, for
+task events see :ref:`user_guide.runtime.task_event_handling`.
 
-   See also :ref:`user_guide.runtime.task_event_handling`.
+.. rubric:: Event Handlers
 
-Workflow event handlers are configured by:
+Workflow events have "handlers" (i.e. hooks) which allow configured commands to
+run when workflow events occur. These can be configured by:
 
 * :cylc:conf:`flow.cylc[scheduler][events]` (per workflow)
 * :cylc:conf:`global.cylc[scheduler][events]` (user/site defaults)
 
+.. rubric:: Abort On Event
 
-Workflow Events
-^^^^^^^^^^^^^^^
+As well as event handlers, you can tell the scheduler to abort (i.e., shut down
+immediately with error status) on certain workflow events, using the
+``abort on ...`` configurations.
+
+.. rubric:: Configuration
+
+Some workflow events have related configurations e.g. for setting the timeout.
+
+.. rubric:: List of workflow events:
+
+.. cylc-scope:: global.cylc[scheduler][events]
+
+.. describe:: startup
+
+   :Event Handler: `startup handlers`
+
+   The scheduler was started or restarted.
+
+   E.G. using one of these commands ``cylc play``, ``cylc vip`` or ``cylc vr``.
+
+.. describe:: shutdown
+
+   :Event Handler: `shutdown handlers`
+
+   The scheduler was shut down.
+
+   E.G. using the ``cylc stop`` command.
+
+.. describe:: abort
 
-The list of events is:
+   :Event Handler: `abort handlers`
 
-startup
-   The scheduler started running the workflow.
-shutdown
-   The workflow finished and the scheduler will shut down.
-abort
    The scheduler shut down early with error status, due to a fatal error
    condition or a configured timeout.
-workflow timeout
+
+.. describe:: workflow timeout
```

Comment:
```suggestion
   condition or a configured timeout.
   
   Note, the `abort` event is not related to "Abort On Event" handlers.

.. describe:: workflow timeout
```

Reply:
@metronnie, resolution to https://github.com/cylc/cylc-doc/pull/685/files#r1478662197

ok?

Reply:
How about "not raised by" instead of "not related to"? But yes that's ok

Reply:
Still need to apply this suggestion

Reply:
Ach, it won't apply, will do manually...